main.cpp

```diff
@@ -823,6 +831,8 @@ static string compile_iteration(const CLIArguments &args, std::vector<uint32_t>
 			msl_comp->add_dynamic_buffer(v.first, v.second, i++);
 		for (auto &v : args.msl_inline_uniform_blocks)
 			msl_comp->add_inline_uniform_block(v.first, v.second);
+		for (auto &v : args.msl_vertex_attribs)
+			msl_comp->add_msl_vertex_attribute(v);
```

Comment:
Is this adding input attributes? I would expect we would need these things for vertex outputs <-> tesc input if anything?

Reply:
I'm using "vertex attributes" during the tessellation control stage to fill in the input struct.

Reply:
"Vertex attributes" have been replaced by "shader inputs."